week-2-practise/level-3/5-indexOf.js

```diff
@@ -4,19 +4,20 @@
 // - insert `newValue` at that index
 // - NOT change the original array
 // - return a new array with the replacement value inserted
-
 function replace(arr, valueToReplace, newValue) {
-  return; // complete this statement
+  var index = arr.indexOf(valueToReplace);
```

Comment:
If the array doesn't contain the given value, it returns `-1`. And you can't use a negative index in an array - so how can you check for `-1` and print a nice error message to the terminal? :)
Extra bonus question!